charts/kkr/wallet-api/templates/ingress/ingress.api.private.yaml

```
@@ -0,0 +1,67 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "wallet-api.fullname" . -}}
+{{- $svcPort := .Values.service.port -}}
+{{- if and .Values.ingress.className (not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion)) }}
+  {{- if not (hasKey .Values.ingress.annotations "kubernetes.io/ingress.class") }}
+  {{- $_ := set .Values.ingress.annotations "kubernetes.io/ingress.class" .Values.ingress.className}}
+  {{- end }}
+{{- end }}
+{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1
+{{- else if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
+{{- else -}}
+apiVersion: extensions/v1beta1
+{{- end }}
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+    {{- include "wallet-api.labels" . | nindent 4 }}
+  annotations:
+    konghq.com/plugins: {{ $fullName }}-cors,{{ $fullName }}-validate-auth-server-token
```

`_helpers.tpl`에서 선언한 내용이 있으니, 이 곳에서도 그 값을 사용하는게 좋을 것 같네요.

```suggestion
    konghq.com/plugins: {{ include "wallet-api.cors" . }},{{ include "wallet-api. validateAuthServerToken" . }}
```